javascript-review.js

```diff
@@ -143,12 +265,42 @@
 // b. 
 // Implement another instance method for your class that takes in age and height (in cms) as arguments (both number type) and adds these arguments as attributes to your person object
 
+class Person {
+  constructor(name) {
+    this.name = name;
+  }
+  sayHi() {
+    console.log('hello');
+  }
+  stats(age, height) {
+    this.age = age;
+    this.height = height;
+  }
+}
+
+const harrison = new Person('harrison');
+console.log(harrison);
+harrison.sayHi()
+
+
 // console.log the updated person object showing all three attributes (name, age, height) as being a part of the person object
+harrison.stats(26, 181);
+console.log(harrison);
 
 // 23. 
 
 // Define a function named waitBeforeSum that takes 2 numbers as arguments. Your function should sum these numbers together but only after waiting for 4 seconds inside of a setTimeout.
 
+// const waitBeforeSum = async (num1, num2) => {
+//   return await setTimeout((num1, num2) => {
+//     return num1 + num2;
+//   }, 4000)
+// }
+// waitBeforeSum(5, 7).then(data => {
+//   console.log(data);
+// })
+
```

Comment:
remember that we can't return a setTimeout, we need to wrap the timeout in a `Promise`